storage/rocksdb/ha_rocksdb.cc

```diff
@@ -11961,23 +11966,31 @@ static void rocksdb_set_max_background_jobs(THD *thd,
   RDB_MUTEX_UNLOCK_CHECK(rdb_sysvars_mutex);
 }
 
+static int rocksdb_validate_update_cf_options(THD *thd,
+                                              struct st_mysql_sys_var *var,
+                                              void *save,
+                                              st_mysql_value *value) {
+  // just disallow setting to null; otherwise same as default string checker
+  char buff[STRING_BUFFER_USUAL_SIZE];
+  const char *str;
+  int length;
+
+  length= sizeof(buff);
+  if ((str= value->val_str(value, buff, &length)))
+    str= thd->strmake(str, length);
+  *(const char**)save= str;
+  return str == nullptr;
```

Comment:
I think allowing str to be set to nullptr is fine. This function should perform the Rdb_cf_options::parse_cf_options(val, &option_map) call though and return success/failure based on the result.